giskard/rag/metrics/ragas_metrics.py

```diff
@@ -26,6 +25,15 @@
     ) from err
 
 
+try:
+    # RAGAS has async functions, we try to apply nest_asyncio to avoid issues in notebooks
+    import nest_asyncio
+
+    nest_asyncio.apply()
+except:  # noqa
```

Comment:
Why an empty exception block here ?

Reply:
just in case `nest_asyncio` is not available or something breaks due to strange env config